src/main/java/calculator/RunCalculator.java

```diff
@@ -0,0 +1,147 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * 계산기
+ */
+public class RunCalculator {
+
+    /**
+     * 계산 시작
+     */
+    public static int runCalculator(final String paramText){
+        paramTextCheck(paramText);
+        String refineText = paramText.replaceAll(" ", "");
+        List<OperatorOrder> list = operatorDivision(refineText);
+        int returnValue = stepToCalculatorRun(refineText, list);
+
+        return returnValue;
+    }
+
+    /**
+     * 계산에 사용될 Text를 Check
+     * @param paramText
+     * @return 정제된 문자열
+     */
+    public static void paramTextCheck(final String paramText){
+        nullOrEmptyCheck(paramText);
+        blankCheck(paramText);
+    }
+
+    private static void nullOrEmptyCheck(String paramText) {
+        if(paramText == null || paramText.length() == 0) {
+            throw new IllegalArgumentException("연산할 Text가 null");
+        }
+    }
+
+    public static void blankCheck(String paramText) {
+        if(paramText.trim().length() == 0) {
```

Comment:
조건문 사용시 다음과 같이 if 와 괄호 사이에 공백을 넣어주시는게 컨벤션이오니 참고 부탁드려요 :)

[7.4절](https://myeonguni.tistory.com/1596)
https://naver.github.io/hackday-conventions-java/#space-between-keyword-parentheses

```suggestion
        if (paramText.trim().length() == 0) {
```